gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,6 @@
-#Mon Mar 20 18:09:37 CET 2017
+#Sat Apr 29 11:24:25 CEST 2017
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-3.4.1-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-3.5-all.zip
```

Comment:
I have no idea...
@DanVanAtta told me to just leave it as it is... https://github.com/triplea-game/triplea/pull/569#r58220703
Eclipse changes this back to bin every time...